app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const getTopics = require("./controllers/topics.controller");
+
+const {
+  handlePSQL400s,
+  handleCustomErrors,
+  handle500Statuses,
+} = require("./controllers/errorHandlers.controllers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
Not needed for this PR, as we're not needing to access to body for a post request, please remove